stdlib/pervasives.mli

```diff
@@ -903,6 +943,11 @@ val really_input_string : in_channel -> int -> string
    characters have been read.
    @since 4.02.0 *)
 
+val input_string_all : in_channel -> string
```

Comment:
@bluddy it doesn't have to be a file :-)
But indeed, `input_lines` would be nice too, it's often very convenient. In the absence of iterators(!), returning a list is acceptable.